components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -743,17 +743,16 @@ private String interpretQueryString (String queryString, Map<String,String[]> pa
             try {
                 value = URLEncoder.encode(value, StandardCharsets.UTF_8.name());
             } catch (UnsupportedEncodingException e) {
-                log.error(e.toString(),e);
+                log.error("Error while encoding the query param: " + value, e);
```

Comment:
shall we add the query param name in the error message?

Reply:
+1